### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.13)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59482]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
When printing a message modal, the second radio button, "Print all messages in this conversation," is followed by a line of text indicating how many messages are in that conversation --- important context for a user deciding how much they want to print. That line of text isn't programmatically associated with the radio button, so it's not announced by a screen reader when a user moves focus to the radio button. Since it's just a `p` paragraph of text that doesn't receive focus, a screen reader user who's tabbing through the interactive elements in the modal is likely to skip right over that text entirely without ever hearing it.

### Link, screenshot or steps to recreate
Example modal body:
```
<div class="modal-body">
   <va-radio enable-analytics="true" class="form-radio-buttons hydrated" role="radiogroup" aria-invalid="true" aria-label="undefined" error="Please select an option to print.">
      <va-radio-option data-testid="radio-print-one-message" label="Print only this message" value="PRINT_MAIN" name="this-message" aria-checked="false" role="radio" id="this-messagePRINT_MAIN" class="hydrated" tabindex="-1"></va-radio-option>
      <va-radio-option data-testid="radio-print-all-messages" aria-label="Print all messages in this conversation (undefined messages)" label="Print all messages in this conversation" value="PRINT_THREAD" name="all-messages" aria-checked="true" role="radio" id="all-messagesPRINT_THREAD" class="hydrated" tabindex="0" style="display: flex;" checked=""></va-radio-option>
   </va-radio>
   <p><strong>(5 messages)</strong></p>
</div>
```

Note that there's no programmatic association between the `p` and the second `va-radio-option`. They appear in sequence, but that's it.

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
This is a perfect use case for the [description text property supported by the component](https://design.va.gov/components/form/radio-button#description-text).

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by lichellebain** [comment]
Jira ticket created:  https://jira.devops.va.gov/browse/MHV-46068

**Comment by oleksii-morgun-va** [comment]
This is being deployed to staging and ready for validation by accessibility

**Comment by shiragoodman** [comment]
@briandeconinck or maybe @AngelaFowler82  - can you please help validate?

**Comment by AngelaFowler82** [comment]
When I use the arrow keys to select the "print all messages" radio button, the screen reader now announces the number of messages in the thread. Closing this as resolved.
